forecast.md

```diff
@@ -7,3 +7,4 @@ Shitty.
 ## Tomorrow
 
 Still shitty but with a chance of meatballs
+MMM.....................meatballs.
```

Comment:
@wxtim  for future reference, i clicked on the plus symbol in the diff to make a comment on this line number
```suggestion
MMM.....................meatballs!
```